plugins/org.jboss.tools.jst.js/src/org/jboss/tools/jst/js/bower/internal/launch/shortcut/GenericNativeBowerLaunch.java

```diff
@@ -44,29 +48,42 @@
 	@Override
 	public void launch(ISelection selection, String mode) {
 		if (selection instanceof IStructuredSelection) {
-			 Object element = ((IStructuredSelection)selection).getFirstElement();
-			 if (element != null && element instanceof IResource) {
+			Object element = ((IStructuredSelection) selection).getFirstElement();
+			if (element != null && element instanceof IResource) {
```

Comment:
good catch ;-) . Frankly never have I seen source code of *instanceof* but after googling found some important info